cli/src/main/kotlin/commands/ReporterCommand.kt

```diff
@@ -174,7 +175,8 @@ class ReporterCommand : CliktCommand(
 
         val copyrightGarbage = copyrightGarbageFile?.readValue<CopyrightGarbage>().orEmpty()
 
-        val packageConfigurationProvider = packageConfigurationOption.createProvider()
+        val packageConfigurationProvider =
```

Comment:
Instead of turning project definition + ort.yml into packageConfigurations, I believe we should instead define an interface
for the License API with getter for exactly what it needs, and then make two implementations for it: one for projects and one for package configurations.

This hides all unneeded things from the logic of the license API, keep package configrations decoupled from the OrtResult and avoids introducing the requirement of being able to map `ort.yml` to `PackageConfigurations`.

Reply:
I've to admit that interface would be identical to package configration for now, so it's not critical.

Reply:
I probably wasn't clear enough, I didn't mean to say that the license resolver should handle these two things different.
My proposal was more like duplicating the PackageConfiguration class, but having it owned by the license API, e.g. similar interface different abstraction level.

Reply:
Ok, I'll leave that for a later refactoring. The part of the code will probably be revisited when fixing the issue with the relative paths of license findings.